src/store/grant/grant.store.ts

```diff
@@ -146,11 +114,8 @@ const actions: ActionTreeAdaptor<Actions, State, RootState> = {
 			});
 	},
 	deleteGrant({ commit, dispatch }, id) {
-		return new Promise(resolve => {
-			setTimeout(() => {
-				resolve(null);
-			}, 750);
-		})
+		return axios
+			.delete<ResponseDataWrapper<Lab>>(`/api/grants/${id}`)
```

Comment:
According to the API, DELETE /api/grants returns only `ResponseBase`.  Please refactor this in lab.store as well. Thank you.